metrics/nodehealthcheck.go

```diff
@@ -71,3 +82,11 @@ func ObserveNodeHealthCheckRemediationDeleted(name, namespace, remediation strin
 		"remediation": remediation,
 	}).Set(0)
 }
+
+func StoreNodeHealthCheckRemediationDuration(name, namespace, remediation string, duration time.Duration) {
```

Comment:
These functions usually start with `Observe...`

Reply:
oh, and as discussed in chat, this is not remediation duration, but node unhealthy duration!

Reply:
for some reason I thought that `Observe` was for gauge only :sweat_smile: 
I'll change it

Reply:
here even the Prometheus method name is "Observe()", see below 😉